src/Components/BlockWithList/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactNode } from 'react'
+
+import {
+  ETheme,
+  widgetThemes,
+} from '../../Constants/Types/theme.types'
+
+import { generateRenderProps } from '../../Utils/helpers'
+
+import './BlockWithList.scss'
+
+export type Props = {
+  /** Define theme */
+  theme?: ETheme
+  frontSide: ReactNode
+  backSide: ReactNode
+}
+
+const defaultProps: Props = {
+  theme: ETheme.DARK,
```

Comment:
I thni we should add props for borderColor, because our mockup has diffrenet mode.